sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -329,62 +330,66 @@ def __init__(self, **kwargs) -> None:
             or self._keyvault_client_configs is not None
             or self._secret_resolver is not None
         )
+        self._current_refresh_check = False
```

Comment:
is this used anywhere?

Reply:
Old property, in a different version of this fix. Missed not removing it.